RUFAS/biophysical/animal/pen.py

```diff
@@ -607,14 +607,14 @@ def get_manure_streams(self) -> dict[str, ManureStream]:
         Returns
         -------
         dict[str, ManureStream]:
-            A dictionary mapping a pen ID to a list of dictionaries. Each dictionary within this
-            list maps a stream name to a `ManureStream` object representing a portion of the pen's total manure.
+            A dictionary mapping stream names to their corresponding ManureStream objects.
 
         Notes
         -----
         - The function first constructs a `total_stream` representing the full manure excretion from a pen.
         - If the animal combination is `LAC_COW`, a portion of this stream is split to a parlor stream
-        based on the `minutes_away_for_milking` ratio using the `split_stream` method.
+        based on the `minutes_away_for_milking` ratio using the `split_stream` method. Parlor deposition
+        is set to 0.0.
```

Comment:
@elle-andreen @tomhuhh it would probably be helpful to have the scientific documentation cited here if possible.